app/chart-history.scss

```diff
@@ -1,45 +1,58 @@
 .ciq-chart-history {
     border-radius: $border-radius;
-    height: 30px;
```

Comment:
This container needs to follow the same `height` as the design

Reply:
The new Figma design required changes. The current dimensions of the container, pixel by pixel, are consistent with the design.

Reply:
There is a height difference between market selector and historical data that needs to follow the design as we cannot bypass QA without fixing it. you may need to find a work around to resolve this.